### PR TITLE
Feat/home page

### PR DESCRIPTION
**Pull request by MatthewArck**
# Changes introduced

CSE-55 and CSE-13

Stubbed out page structure and is awaiting integration with other components.

## Related issue(s):

- CSE-_user story or bug_
- CSE-_task number_

## Contributor checklist

- [ ] Wrote/updated tests for introduced changes
- [ ] Added new stories for created/modified components (if applicable)
- [ ] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [ ] Verified that there are no issues in CircleCI
- [ ] Assigned yourself to the PR
- [ ] Removed `WIP:` from the PR title
- [ ] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

**Review by tnoelcke — CHANGES_REQUESTED** [review]
We need to remove the Coverage files and Reports directory form this PR. Those are in the git ignore and shouldn't have gotten checked in in the first place.

File: src/clients/search/index.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+export interface ISearchClient {
+  search: (params: ISearchParams) => Promise<IClaim | Error>;
+}
+
+/**
+ * Cliuent class that cxommunicates with the cse api
+ * @class {SearchClient}
+ */
+export class SearchClient implements ISearchClient {
```
Comment (danvanhorn) [inline src/clients/search/index.ts:15]: this code must be from branching off of my earlier branch, I would say that it is not ready to be merged yet but i know that there are 6 other PRs active and some of the other developers have been refactoring this code. considering the time frame we're working with, I think we should merge this in.

File: src/clients/search/search.spec.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('search client', () => {
```
Comment (danvanhorn) [inline src/clients/search/search.spec.ts:1]: Once again, obviously incomplete but in the interest of time we should go with this.

**Review by danvanhorn — CHANGES_REQUESTED** [review]
I don't know what tasks are associated with this PR, I assume it is CSE-68 or CSE-307, which are duplicate tasks concerned with "implementing home page design". There is another task, CSE-71, which is about connecting the home buttons to the relevant pages. From reading that task description and cross-referencing it with the code, I don't think this is implemented. I need more information about the related task numbers and a description of the changes if I'm going to be able to review this properly.

**Pull request merged**